tests/tests_suite_test.go

```diff
@@ -34,6 +38,36 @@ var _ = BeforeSuite(func() {
 		err := tests.CreateNamespaceWithParameter(v)
 		Expect(err).ToNot(HaveOccurred())
 	}
+	// Create a directory and clone the ansible-kubevirt-module in it
+	os.Mkdir(ansibleModuleClonePath, 0755)
+	_, err := git.PlainClone(ansibleModuleClonePath, false, &git.CloneOptions{
+		URL:      ansibleModuleURL,
+		Progress: os.Stdout,
+	})
+	Expect(err).ToNot(HaveOccurred())
+
+	// install Ansible from Github devel
+	ansible, ansible_err := exec.Command("pip", "install", "git+https://github.com/ansible/ansible.git", "--user").Output()
+	if ansible_err != nil {
+		fmt.Println(ansible_err)
+		fmt.Println(ansible)
+		Expect(ansible_err).ToNot(HaveOccurred())
+	} else {
+		fmt.Printf("\nInstalled ansible\n")
```

Comment:
Try to print out `ansible` stdout as well.

Reply:
done